scripts/app-config-replace.js

```diff
@@ -21,6 +21,7 @@ const options = {
       appConfig.apiHost = API_HOST;
       appConfig.bpmHost = API_PROCESS_HOST;
       appConfig.ecmHost = API_CONTENT_HOST;
+      appConfig.baseShareUrl = API_CONTENT_HOST + '/#/preview/s';
       appConfig.aosHost = API_AOS_HOST;
```

Comment:
would have to be `appConfig.aosHost = API_AOS_HOST + '/alfresco/aos'` to actually work, but it's not used by any tests at the moment afaik

Reply:
I reckon aosHost should contain everything, `url` and `path` as well, this is why we moved this data into env vars. I believe, a code file should contain only behaviour and not data. For data, we can simply put them into environment variables.

Reply:
I added this comment after I saw [this](https://github.com/Alfresco/alfresco-content-app/pull/1669/files#diff-354f30a63fb0907d4ad57269548329e3R46). This makes it necessary to add the path in here, as far as I can understand.

Reply:
Sorry, I'm not sure I can follow you. Why is it necessary to add this path in this codefile and not as part of an environment variable?

Reply:
this should be set by an environment variable, as you say. But at the moment, in travis.yml this value is `- API_AOS_HOST=$API_CONTENT_HOST_REMOTE`
So, considering the env variable used on travis, then the path needs to be added  like a mentioned above. It's either this or correctly set the variable on travis.yml 😃